internal/systemd.go

```diff
@@ -0,0 +1,30 @@
+package internal
+
+import (
+	"os/exec"
+	"strings"
+)
+
+// IsServiceActive check if given service is active
+func IsServiceActive(service string) bool {
+	out, err := exec.Command(SystemctlExec, "is-active", service).Output()
```

Comment:
If we enable access to `org.freedesktop.systemd1.Manager` anyway, maybe the better approach is to check the service status directly with dbus message, instead executing the command line binary?

Reply:
This is legacy code, just moved it from other file into new file. Will check how to do that via dbus.

Reply:
Implemented direct dbus calls - not sure if it is better, but seems more complex in terms of code size.